test/clientTest.js

```diff
@@ -19,7 +19,7 @@ mc.supportedVersions.forEach(function(supportedVersion,i) {
   const PORT=Math.round(30000+Math.random()*20000);
   const mcData = require("minecraft-data")(supportedVersion);
   const version = mcData.version;
-  const MC_SERVER_JAR_DIR = process.env.MC_SERVER_JAR_DIR;
+  const MC_SERVER_JAR_DIR = process.env.MC_SERVER_JAR_DIR || '/tmp';
```

Comment:
You can use [`os.tmpdir()`](https://nodejs.org/api/os.html#os_os_tmpdir) to get the system's temp directory. I'd like it if this used it instead. On OSX, the tmp dir is something like `/var/folders/_1/<some random string>/T`, and on windows it's `\\temp`. Letting the stdlib take care of these kind of discrepency seems like a good idea :)

Reply:
Good catch. Made it use os.tmpdir().